plugins/modules/zos_mvs_raw.py

```diff
@@ -3090,6 +3097,49 @@ def get_content(formatted_name, binary=False, from_encoding=None, to_encoding=No
         return stdout
 
 
+def modify_contents(contents):
+    """Return the content of dd_input to a valid form for a JCL program.
+
+    Args:
+        contents (str or list): The string or list with the program.
+
+    Returns:
+        contents: The content in a proper multi line str.
+    """
+    if isinstance(contents, list):
+        contents = "\n".join(contents)
+        contents = add_space(contents)
+    else:
+        if contents[0] != " " or contents[0] != "-":
+            contents = "  {0}".format(contents)
+        contents = list(contents.split("\n"))
+        contents = add_space(contents)
+        contents = "\n".join(contents)
+    return contents
+
+
+def add_space(lines):
+    """Return the array with two spaces at the beggining.
+
+    Args:
+        lines (list): The list with a line of a program.
+
+    Returns:
+        lines: The list in a proper two spaces and the code.
+    """
+    module = AnsibleModule
+    for line in lines:
+        if len(line) == 0:
+            pass
+        else:
+            if line[0] != " " or line[0] != "-":
+                if len(line) > 78:
+                  module.fail_json(msg="Length of the line {0} over 80, dataset can not be written".format(line))
+                else:
+                  line = "  {0}".format(line)
```

Comment:
I would prefer this addition to be a "smart" addition by only adding if the first two chars are not empty spaces, but will leave this to your consideration, not a deal breaker.